6.2 versions fail on build with typescript / missing semver in type declarations.

### 🐛 Description

I have a problem when trying to use the latest (and maybe earlier) version of the magic sdk for the browser. I get the following errors after running with 6.2 during compilation (typescript, angular). Previous version that I know it worked was 5.x:


```
Error: node_modules/@magic-sdk/provider/dist/types/util/index.d.ts:5:25 - error TS2307: Cannot find module './semver' or its corresponding type declarations.

5 export * as semver from './semver';
```

### 🧩 Steps to Reproduce

1. Create a project that uses typescript (and maybe angular). 
2. add magic 6.2.

### 🤔 Expected behavior

It should compile

### 😮 Actual behavior

```
Error: node_modules/@magic-sdk/provider/dist/types/util/index.d.ts:5:25 - error TS2307: Cannot find module './semver' or its corresponding type declarations.

5 export * as semver from './semver';
```


### 🌎 Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| `magic-sdk`      | 6.x
| Browser          |  n/a
| `yarn`           | npm 6.14.11
| Operating System | Mac

--- a/packages/@magic-sdk/provider/src/util/index.ts
+++ b/packages/@magic-sdk/provider/src/util/index.ts
@@ -2,7 +2,6 @@ export * from './base64-json';
 export * from './events';
 export * from './get-payload-id';
 export * from './promise-tools';
-export * as semver from './semver';
 export * as storage from './storage';
 export * from './type-guards';
 export * from './url';

